.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I fully agree that having it (at least) here as an example is definitely what we should do. I don't know it just feels wrong having it in the repo, but then I guess it comes down to the same thing that including build output does, convenience.